qiskit_ibm_provider/ibm_backend.py

```diff
@@ -458,6 +458,12 @@ def run(
         """
         # pylint: disable=arguments-differ
         validate_job_tags(job_tags, IBMBackendValueError)
+        if hasattr(self, "backend_status"):
```

Comment:
I am not sure how and where backend_status is set in self?

Reply:
oops, meant it to be `hasattr(self._api_client, "backend_status")` so the unit tests that use `BaseFakeAPI` pass 

fixed now